examples/aggregated_k8s/agg.yaml

```diff
@@ -201,38 +151,35 @@ spec:
           value: "Qwen/Qwen3-0.6B"
         - name: MODEL_CACHE_PATH
           value: "/model/.model-express/cache"
+        - name: HF_HUB_CACHE
+          value: "/model/.model-express/cache"
+        - name: MODEL_EXPRESS_URL
+          value: "http://vllm-agg-modelexpress:8000"
+        - name: VLLM_GPU_MEMORY_UTILIZATION
+          value: "0.95"
+        - name: VLLM_MAX_MODEL_LEN
+          value: "8192"
       extraPodSpec:
         mainContainer:
           startupProbe:
             httpGet:
               path: /health
               port: 9090
-            periodSeconds: 10
-            failureThreshold: 60
-          image: nvcr.io/nvstaging/ai-dynamo/vllm-runtime:0.4.0
+            periodSeconds: 30
+            failureThreshold: 240
+          image: nvcr.io/nvidia/ai-dynamo/vllm-runtime:my-tag
           workingDir: /workspace/components/backends/vllm
           command:
             - /bin/sh
             - -c
           args:
             - |
-              echo "Waiting for model to be ready..."
-              # Convert model name to cache directory format (e.g., "Qwen/Qwen3-0.6B" -> "models--Qwen--Qwen3-0.6B")
-              MODEL_CACHE_DIR=$(echo $MODEL_NAME | sed 's/\//--/g')
-              MODEL_CACHE_DIR="models--$MODEL_CACHE_DIR"
-
-              for i in {1..120}; do
-                if [ -L "$MODEL_CACHE_PATH/$MODEL_CACHE_DIR/snapshots/latest" ] && [ -d "$MODEL_CACHE_PATH/$MODEL_CACHE_DIR/snapshots/latest" ]; then
-                  echo "Model is ready! Starting VLLM worker..."
-                  break
-                fi
-                echo "Waiting for model... ($i/120)"
-                sleep 5
-              done
-
-              echo "Model path: $MODEL_CACHE_PATH/$MODEL_CACHE_DIR/snapshots/latest/"
-              ls -la $MODEL_CACHE_PATH/$MODEL_CACHE_DIR/snapshots/
-              python3 -m dynamo.vllm --model $MODEL_CACHE_PATH/$MODEL_CACHE_DIR/snapshots/latest/ | tee /tmp/vllm.log
-      pvc:
+              python3 -m dynamo.vllm --model $MODEL_NAME --tensor-parallel-size 1 | tee /tmp/vllm.log
+          resources:
+            requests:
+              ephemeral-storage: "50Gi"
+            limits:
+              ephemeral-storage: "50Gi"
+      pvcs:
```

Comment:
I'm not expert in k8s CRD, but is this the right syntax for DGD? The [api reference](https://github.com/ai-dynamo/dynamo/blob/main/docs/kubernetes/api_reference.md#dynamographdeploymentspec) seems to define `pvcs` along the `services` level so I used to define it like following:
```
spec:
  pvcs:
    - name: model-cache-pvc
  services:
    SomeService:
      ...
      volumeMounts:
        - name: model-cache-pvc
          mountPoint: /model
```
maybe both works. Wanted to make sure we're both doing it right.
cc @biswapanda @julienmancuso

Reply:
hi @hutm, are you referring to my example in the comment or Kavin's agg.yaml in the PR?